javascript-and-json/main.js

```diff
@@ -0,0 +1,37 @@
+var books = [
+  {
+    isbn: '123',
+    title: 'JavaScript',
+    author: 'Miori'
+  },
+  {
+    isbn: '456',
+    title: 'HTML',
+    author: 'Kazuma'
+  },
+  {
+    isbn: '789',
+    title: 'CSS',
+    author: 'Yuki'
+  }
+];
+
+console.log('books:', books);
+console.log('typeof books:', typeof books);
+
+var booksInJson = JSON.stringify(books);
+console.log('JSON string of books:', booksInJson);
+console.log('typeof booksInJson:', typeof booksInJson);
+
+var student = {
+  id: '1234',
+  name: 'Miori'
+};
+
+var studentInJson = JSON.stringify(student);
```

Comment:
instead of using `stringify` here, you are asked to manually create a JSON string